oh_connection/templates/oh_connection/index.html

```diff
@@ -4,7 +4,7 @@
 
       <div class="jumbotron">
         <img alt="Brand" src="{% if config.logo_url %}{{config.logo_url}}{% else %}/static/default_logo.png{% endif %}" height="150px" align="left">
-        <h1>Welcome to {{config.project_title}}</h1>
+        <h1>Welcome to {{ config.project_title|default:"My Open Humans Project" }}</h1>
         <p>
           {{config.project_description}}
```

Comment:
It would be great if you could add a `default:` for the `config.project_description` too, as these are now also allowed to be empty. :-)